src/libs/actions/Report.js

```diff
@@ -379,11 +379,26 @@ function setLocalLastRead(reportID, sequenceNumber) {
 
     // Update the report optimistically
     Onyx.merge(`${ONYXKEYS.COLLECTION.REPORT}${reportID}`, {
-        unreadActionCount: 0,
+        unreadActionCount: Math.max(reportMaxSequenceNumbers[reportID] - sequenceNumber, 0),
```

Comment:
Please add a code comment to explain why this logic is this way (I don't understand it myself from just looking at it).